core/src/main/java/io/questdb/std/Numbers.java

```diff
@@ -652,6 +652,9 @@ public static int getIPv4Netmask(CharSequence sequence) {
 
```

Comment:
Isn't `/31` a valid netmask ? 

For example :
```sql
select * from tipv4 where '255.255.255.128/31' >>= ip;
```
returns 
```sql
invalid argument: 255.255.255.128/31
```

In PostgreSQL it works fine.